semgrep/semgrep/rule_match.py

```diff
@@ -109,13 +111,20 @@ def lines(self) -> List[str]:
 
         Assumes file exists.  Note that start/end line is one-indexed
         """
-        if "lines" in self.extra:
-            return self.extra["lines"]
+        start_line = self.start["line"] - 1
+        end_line = self.end["line"]
```

Comment:
Suggestion:

Rename this to `line_after_end` or also subtract 1

Reply:
Added a comment addressing - it's kinda confusing with the `islice` call below :+1: